docs/sections/user_guide/cli/mode_rocoto.rst

```diff
@@ -48,7 +53,155 @@ The ``uw`` mode for realizing and validating Rocoto XML documents
 Examples
 ~~~~~~~~
 
-TBD
+The examples that follow use a Rocoto YAML file ``rocoto.yaml`` with content
```

Comment:
This may be a philosophical distinction, but I'd refer to the YAML file as a "UW YAML" file instead of a "Rocoto YAML" file, since the YAML spec is defined (and enforced) by UW; and because the Rocoto-related configuration could -- and I assume eventually will -- coexist with config blocks for other components in the same UW YAML file.